src/main/java/seedu/duke/command/ListNoteCommand.java

```diff
@@ -248,13 +251,37 @@ public StringBuilder getSortedString(ArrayList<Note> sortedNotes) {
      */
     public StringBuilder getNoteString(ArrayList<Note> notesList) {
         StringBuilder noteString = new StringBuilder();
+        int j = 1;
 
         for (int i = 0; i < notesList.size(); i++) {
-            noteString.append(i + 1).append(SUFFIX_INDEX)
-                    .append(notesList.get(i).getTitle())
-                    .append(" ")
-                    .append(notesList.get(i).getTagsName())
-                    .append(Formatter.LS);
+            if (!notebook.getNotes().get(i).getIsArchived()) {
+                noteString.append(j).append(SUFFIX_INDEX)
+                        .append(notesList.get(i).getTitle())
+                        .append(" ")
+                        .append(notesList.get(i).getTagsName())
+                        .append(Formatter.LS);
+
+                j++;
+            }
+        }
+
+        return noteString;
+    }
+
+    public StringBuilder getArchiveString() {
+        StringBuilder noteString = new StringBuilder();
+        int j = 1;
+
+        for (int i = 0; i < notebook.getSize(); i++) {
+            if (notebook.getNotes().get(i).getIsArchived()) {
+                noteString.append(j).append(SUFFIX_INDEX)
+                        .append(notebook.getNote(i).getTitle())
+                        .append(" ")
+                        .append(notebook.getNote(i).getTagsName())
+                        .append(Formatter.LS);
+
+                j++;
+            }
```

Comment:
All formatting should be inside formatter class. Can change the formatNotebook to formatNotes(arrayList Notes)